tensorflow/python/kernel_tests/math_ops/aggregate_ops_test.py

```diff
@@ -61,8 +61,10 @@ def testAddN(self):
           actual = self.evaluate(math_ops.add_n(data))
           expected = np.sum(np.vstack(
               [np.expand_dims(d, 0) for d in data]), axis=0)
-          tol = 5e-3 if dtype == dtypes.float16 else 5e-7
-          self.assertAllClose(expected, actual, rtol=tol, atol=tol)
+          self.assertAllCloseAccordingToType(
```

Comment:
Eek, do we really need to specify all of these?  Ideally the defaults would just work.  At the least, you shouldn't need to specify the same values for both `atol` and `float_atol`

Reply:
Verified that the tests pass with default values; updated it.